app/app.rb

```diff
@@ -0,0 +1,86 @@
+require 'sinatra/base'
+require 'json'
+
+class FakeIdvaas < Sinatra::Application
+  get '/' do
+    'Hello, USA!'
+  end
+
+  get '/idvaas/user/:user_uuid/question/?' do
```

Comment:
I don't believe we need the trailing `?`. For this first iteration it should be sufficient to return a single question that makes the pass/fail route obvious. It also needs to match the routes used by the IdP:

```
  def question
    response = IdpRequest.new("/user/#{current_user.uuid}/question/").response
    ...
  end

  def answer
    response = IdpPostRequest.new("/user/#{current_user.uuid}/question/", answer_data).response
    ...
  end
```